content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide--Configure_YubiHSM_2_Key_Storage_Provider_for_Microsoft_Windows_Server/Getting_Help.adoc

```diff
@@ -2,13 +2,13 @@
 
 Should you require assistance when using this guide to deploy YubiHSM 2 on Windows, start by referencing the product documentation and currently known issues:
 
-https://developers.yubico.com/[Yubico Developers website]
+link:../../../[Yubico Developers website]
 
 https://support.yubico.com/support/home[Yubico Support]
 
-https://developers.yubico.com/YubiHSM2/[YubiHSM 2 Product Documentation]
+link:../[YubiHSM 2 Product Documentation]
 
-https://developers.yubico.com/YubiHSM2/Releases/Known_issues.html[Known Issues and Limitations]
+link:../Releases/Known_issues.adoc[Known Issues and Limitations]
```

Comment:
This needs another ../ to land on the right page

Reply:
https://github.com/Yubico/developers.yubico.com/pull/249/commits/fc45cd8517b211d9bc7d7e5f61d83219294579bc